x-pack/plugins/whitelabelling/config.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export const configSchema = schema.object({
```

Comment:
`config-schema`'s schemas shouldn't be in `common` code (else `config-schema` implementation will leak inside the browser bundle, adding a bunch 100kos). This needs to be moved to the `server` part of the plugin. (if the `ConfigSchema` type needs to be used on the browser-side, just explicitly define it instead of inferring it via `TypeOf<typeof ...>`)